subsys/bluetooth/host/conn.c

```diff
@@ -1884,6 +1896,24 @@ int bt_conn_get_info(const struct bt_conn *conn, struct bt_conn_info *info)
 		info->le.interval = conn->le.interval;
 		info->le.latency = conn->le.latency;
 		info->le.timeout = conn->le.timeout;
+		info->le.features = conn->le.features;
+#if defined(CONFIG_BT_REMOTE_VERSION)
+		if (atomic_test_bit(conn->flags, BT_CONN_HAVE_VERSION_INFO)) {
+			info->version = conn->rv.version;
+			info->manufacturer = conn->rv.manufacturer;
+			info->subversion = conn->rv.subversion;
+		} else {
+			int err = hci_le_read_remote_version(conn);
+
+			if (err) {
+				return err;
+			}
+
+			info->version = conn->rv.version;
+			info->manufacturer = conn->rv.manufacturer;
+			info->subversion = conn->rv.subversion;
+		}
+#endif /* CONFIG_BT_REMOTE_VERSION */
```

Comment:
Sorry for not realising this earlier, but this whole block should apply both for BR/EDR connections as well as LE connections, so pleas move it outside (before) the switch statement.